miner/src/block_assembler.rs

```diff
@@ -413,6 +416,16 @@ impl BlockAssembler {
         Ok(template)
     }
 
+    fn package_txs(
+        &self,
+        chain_state: &ChainState,
+        size_limit: usize,
+        cycles_limit: Cycle,
+    ) -> (Vec<TxEntry>, usize, Cycle) {
+        let tx_pool = chain_state.tx_pool();
+        CommitTxsScanner::new(tx_pool.proposed()).transactions_to_commit(size_limit, cycles_limit)
```

Comment:
Not require, just comment, maybe we should uniform `txs` `transactions`, it's weird seeing they appear both.

Reply:
Yeah, forgot to rename the function.